### PR TITLE
AFC-1306 [ADD] partner_test-data_fenix : Code moved from fenix_test_contatcts

### PR DESCRIPTION
**Pull request by arturchilangwa-vertel**
This new module is only a renames fenix_test_contacts

### CODE REVIEW

File: partner_test-data_fenix/data/res_company.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <data noupdate="1">
+        <record id="jack_sons_company" model="res.company">
```
Comment (nils-vertel) [inline partner_test-data_fenix/data/res_company.xml:3]: jack_sons_company?
Is this test data in some way?

File: partner_test-data_fenix/data/res_partner.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<odoo>
```
Comment (nils-vertel) [inline partner_test-data_fenix/data/res_partner.xml:1]: Is this test data?

File: partner_test-data_fenix/data/res_user.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<odoo>
```
Comment (nils-vertel) [inline partner_test-data_fenix/data/res_user.xml:1]: Same here, is this testdata?

**Review by nils-vertel — CHANGES_REQUESTED** [review]
Please verify that the data is correct, it looks good and could be approved if the data is correct.

File: partner_test-data_fenix/data/res_company.xml (hunk 1 above)
Comment (arturchilangwa-vertel) [reply partner_test-data_fenix/data/res_company.xml:3]: Yes, it's the raw data that defines an instance of a the test models. The same with the same with res_users.xml & res_partner.xml. I tried it on a empty DB and it works!

**Pull request merged**